TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Administration/UsersController.php

```diff
@@ -295,4 +303,18 @@ protected function assignElectronicAddressOptions()
             'electronicAddressUsageTypes' => $electronicAddressUsageTypes
         ));
     }
+
+    /**
+     * TODO: With Flow 4.0 replace getting providers with access over AuthenticationManagerInterface
```

Comment:
Wouldn't this change be released at the same time as: https://github.com/neos/flow-development-collection/pull/404 
I suppose both changes would be merged into the next mayor release, so you can already use your getter here.